{"uuid": "65e48401-92cf-45a7-9e97-898b2d599326", "children": ["e72f33d5-ca51-4016-b963-d3b739f105da", "ee23f34d-d747-49ad-b0f4-bba0ecfd63e3", "84afd5f5-7b4e-4f9e-ae9d-91e7ef146a44"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "NameError: name 'login' is not defined\n", "trace": "  File \"C:\\Program Files\\python\\python36\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Program Files\\python\\python36\\lib\\site-packages\\_pytest\\fixtures.py\", line 963, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Program Files\\python\\python36\\lib\\site-packages\\_pytest\\fixtures.py\", line 787, in call_fixture_func\n    res = next(it)\n  File \"C:\\Program Files\\python\\python36\\lib\\site-packages\\_pytest\\python.py\", line 703, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"C:\\Program Files\\python\\python36\\lib\\site-packages\\_pytest\\python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"C:\\Users\\Administrator\\Desktop\\apiauto-zhuosui1.5\\TestCase\\MarketQuotation\\test_conceptlist.py\", line 15, in setup_class\n    login()\n"}, "start": 1592559500276, "stop": 1592559500276}], "start": 1592559500276, "stop": 1592559500358}